### PR TITLE
Fixed ckeditor mentions

### PR DESCRIPTION
**Pull request by rustam-miagdeev**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -238,7 +238,7 @@
         "drupal/bootstrap4_modal": "^2.0",
         "drupal/bootstrap_barrio": "^5.0",
         "drupal/ckeditor_media_embed": "^1.9",
-        "drupal/ckeditor_mentions": "2.x-dev#272a28e7",
+        "drupal/ckeditor_mentions": "^2.0@RC",
```
Comment (heyyo-droid) [inline composer.json]: better not to use ^ but only exact version.

Comment (heyyo-droid) [reply composer.json]: 8.x-2.0-rc1 is the last release to still support PHP 7.x

File: composer.json [hunk 2]
```diff
@@ -348,9 +348,6 @@
             "drupal/rules": {
                 "ServiceNotFoundException when installing a rule via configuration": "https://www.drupal.org/files/issues/2020-07-10/ServiceNotFoundException-when-installing-a-rule-via-configuration.2870851-10_0.patch"
             },
-            "drupal/ckeditor_mentions": {
```
Comment (heyyo-droid) [inline composer.json:351]: Why do we remove this patch ?

File: composer.json (hunk 1 above)
Comment (rustam-miagdeev) [reply composer.json]: Fixed

File: composer.json (hunk 2 above)
Comment (rustam-miagdeev) [reply composer.json:351]: The 2.0@RC version contains this patch.

**Pull request merged**